src/database/seeders/20190916205822-demo-request.js

```diff
@@ -12,6 +12,56 @@ export default {
         user: 5,
         createdAt: new Date(),
         updatedAt: new Date()
+      },
+      {
+        from: 'ACCRA, GHANA',
+        travelDate: ['2019-08-01'],
+        returnDate: '2019-12-12',
+        reason: 'Business Travel',
+        status: 'Approved',
+        user: 3,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        from: 'MANILLA, PHILIPINES',
+        travelDate: ['2019-07-01'],
+        returnDate: '2019-12-12',
+        reason: 'Business Travel',
+        status: 'Rejected',
+        user: 5,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        from: 'NAKURU, KENYA',
+        travelDate: ['2019-04-01', '2019-05-21'],
+        returnDate: '2019-07-12',
+        reason: 'Still figuring this out',
+        status: 'Approved',
+        user: 3,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        from: 'VICTORIA, EAFRICA',
+        travelDate: ['2019-09-01', '2019-10-01'],
+        returnDate: '2019-06-12',
+        reason: 'i want to checkout the nile crocdiles',
+        status: 'Approved',
+        user: 3,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
```

Comment:
Just update this seeder. The travelDate is greater than request date

Reply:
have worked on it